contracts/staking/StakeManager.sol

```diff
@@ -41,6 +41,7 @@ contract StakeManager is Validator, IStakeManager, RootChainable, Lockable {
   uint256 public NFTCounter = 1;
   uint256 public totalRewards;
   uint256 public totalRewardsLiquidated;
+  uint256 public auctionInterval = DYNASTY.div(4); // 1 week in epochs
```

Comment:
Did you want to use, `auctionPeriod` here?
From the spec,
```
AuctionPeriod: Number of blocks for which the on-chain auction lasts.
AuctionInterval: Number of blocks after which auction is held.
```

Reply:
yeah good catch switch of name. THanks